dev/MRTCore/TestApps/winui_desktop_packaged_app_cpp/winui_desktop_packaged_app_cpp/App.cpp

```diff
@@ -41,7 +41,7 @@ App::App()
 
 App::~App()
 {
-    MrmDestroyResourceManager(m_resourceManagerMrm);
+    //MrmDestroyResourceManager(m_resourceManagerMrm);
```

Comment:
The WinUI package does not contain the C headers, so this is disabled until we move to the Reunion feed.

Reply:
Would be good to add this as a comment here and in the other places with commented out code.

Reply:
+1, prefer that that has the form of a comment rather than just a commented out line.